src/init/features/firestore.ts

```diff
@@ -1,33 +1,31 @@
-"use strict";
-
-var clc = require("cli-color");
-var fs = require("fs");
-
-var FirebaseError = require("../../error");
-var gcp = require("../../gcp");
-var iv2 = require("../../firestore/indexes");
-var fsutils = require("../../fsutils");
-var prompt = require("../../prompt");
-var logger = require("../../logger");
-var utils = require("../../utils");
-var requireAccess = require("../../requireAccess");
-var scopes = require("../../scopes");
-
-var indexes = new iv2.FirestoreIndexes();
-
-var RULES_TEMPLATE = fs.readFileSync(
+import clc = require("cli-color");
+import fs = require("fs");
+
+import FirebaseError = require("../../error");
+import gcp = require("../../gcp");
+import iv2 = require("../../firestore/indexes");
+import fsutils = require("../../fsutils");
+import prompt = require("../../prompt");
+import logger = require("../../logger");
+import utils = require("../../utils");
+import requireAccess = require("../../requireAccess");
+import scopes = require("../../scopes");
+
+const indexes = new iv2.FirestoreIndexes();
+
+const RULES_TEMPLATE = fs.readFileSync(
   __dirname + "/../../../templates/init/firestore/firestore.rules",
   "utf8"
 );
 
-var DEFAULT_RULES_FILE = "firestore.rules";
+const DEFAULT_RULES_FILE = "firestore.rules";
 
-var INDEXES_TEMPLATE = fs.readFileSync(
+const INDEXES_TEMPLATE = fs.readFileSync(
   __dirname + "/../../../templates/init/firestore/firestore.indexes.json",
   "utf8"
 );
 
-var _initRules = function(setup, config) {
+async function _initRules(setup: any, config: any): Promise<any> {
```

Comment:
I would ask that any "private" functions (that begin with an underscore) have the underscore removed. Since they're not exported, they're not public, and that should be the delineating factor.

Reply:
done